File: 프로그래머스/lv1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.py
def solution(left, right):
    ans = 0
    for i in range(left, right + 1):
        y = 0
        for j in range(1, i + 1):
            if i % j == 0:
                y += 1
        if y % 2 == 0:
            ans += i
        else:
            ans -= i
    return ans

File: 프로그래머스/lv1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.js
function solution(left, right) {
  let answer = 0;
  for (let i = left; i <= right; i++) {
    y = 0;
    for (let j = 1; j <= i; j++) {
      if (i % j == 0) {
        y += 1;
      }
    }
    if (y % 2 == 0) {
      answer += i;
    } else {
      answer -= i;
    }
  }
  return answer;
}